### ISSUE TITLE
Pull from Membership List to Sync to Firebase

### ISSUE THREAD

**Issue by midcenturyminnie** [dalenguyen/firebase-wordpress-plugin#17]
Hi! This may be a dead end, but I thought I'd ask.

I'm working on building a website to correspond with an App built within Firebase. Clients come to our site, purchase the membership, create their login, and then download the app - sign in and voila. We have a membership area on the website with member-only content. Is there a way for your Plugin to pull directly from a Membership plugin, so that every new member gets auto-synced with the Firebase database?

Thanks!

**Comment by alexmasita** [comment]
This would be a great enhancement.  I would like to also know whether by logging to wordpress via firebase authentication allows you to access authenticated wordpress content.

**Comment by ErisEnterprise** [comment]
@alexmasita, is there any update to this thread?

**Comment by alexmasita** [comment]
Hi @ErisEnterprise.  Not that I know of.  But I have since moved on to explore other solutions.  The idea that I am going to experiment with is the one that uses single sign-on for multiple apps.  This is where I found a WordPress plugin that allows you to log in via firebase authentication and automatically creates a user in Wordpress if not created: [Firebase Authentication](https://wordpress.org/plugins/firebase-authentication/).

It still feels a bit hacky as you have two tables of users, one in firebase & the other in Wordpress.  So I am still looking for a true single sign-on solution where you have a central database server of users and applications check with that server where if the user is verified to have logged on the central server, a token is issued to all registered applications without password prompts.
